user_docs/en/userGuide.md

```diff
@@ -2297,6 +2297,12 @@ If this is enabled, NVDA will automatically speak the character at the cursor wh
 
 To toggle this option from anywhere, please assign a custom gesture using the [Input Gestures dialog](#InputGestures).
 
+##### Speak character when navigating by line or paragraph {#BrailleSpeakOnNavigating}
+
+If this is enabled, NVDA will automatically speak the current line or paragraph when navigating to it with braille navigation keys.
```

Comment:
Slight rewording that seems better to me, although that cans probably still be improved.

```suggestion
If this is enabled, NVDA will speak the reached line or paragraph when moving the braille display to the previous or next line.
```